app/realtime.php

```diff
@@ -32,41 +41,81 @@
 ini_set('default_socket_timeout', -1);
 Swoole\Runtime::enableCoroutine(SWOOLE_HOOK_ALL);
 
-$server = new Server("0.0.0.0", 80);
+$server = new Server('0.0.0.0', 80);
+
+$server->set([
+    'websocket_compression' => true,
```

Comment:
Which type of compression is being used here?

Reply:
iirc, it enables support for: https://tools.ietf.org/html/rfc7692